api/agent/drivers/docker/docker_client.go

```diff
@@ -227,6 +229,24 @@ func filterNoSuchContainer(ctx context.Context, err error) error {
 	return err
 }
 
+func (d *dockerWrap) LoadImages(ctx context.Context, filePath string) error {
```

Comment:
Based upon our offline discussion, would you be willing to add a comment here that explains the intricacies of save/load and how we need to ensure that the saved image that we load actually bears a tag?

Reply:
OK, added a link to docker-save documentation in options.md. Docker-save documentation shows how to cherry pick tags.